From 76f850333d3f43c5d2a18ac3f7487d09f8476115 Mon Sep 17 00:00:00 2001
From: gbhakteX <gurudattax.bhakte@intel.com>
Date: Fri, 27 Jul 2012 11:27:58 +0530
Subject: Latency optimization in bufferCount / MW (Audio android framework).

BZ: 26849

BufferCount for AudioSink/AudioTrack is changed to 2 from 4. This
is done to improve the latency for the audio buffer.

Category: feature differentiation
Domain: AUDIO_MEDIA-LPA
Origin: Internal
Upstream-Candidate: no, propreitary

Orig-Change-Id: I7c5845b9e97513c46c2d8886d31169f417a8844b

Change-Id: Ic14d0b135e35dd2c8e39f962f7e83cc9aad730ab
Signed-off-by: gbhakteX <gurudattax.bhakte@intel.com>
Reviewed-on: http://android.intel.com:8080/77920
Reviewed-by: Yao, Yong <yong.yao@intel.com>
Reviewed-by: Kandasamy, Muthukumar <muthukumar.kandasamy@intel.com>
Reviewed-by: M, Arulselvan <arulselvan.m@intel.com>
Tested-by: M, Arulselvan <arulselvan.m@intel.com>
Reviewed-by: cactus <cactus@intel.com>
Tested-by: cactus <cactus@intel.com>
Signed-off-by: Arulselvan M <arulselvan.m@intel.com>
---
 include/media/MediaPlayerInterface.h |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/include/media/MediaPlayerInterface.h b/include/media/MediaPlayerInterface.h
index cc244f0..f002a64 100644
--- a/include/media/MediaPlayerInterface.h
+++ b/include/media/MediaPlayerInterface.h
@@ -53,7 +53,7 @@ enum player_type {
 };
 
 
-#define DEFAULT_AUDIOSINK_BUFFERCOUNT 4
+#define DEFAULT_AUDIOSINK_BUFFERCOUNT 2
 #define DEFAULT_AUDIOSINK_BUFFERSIZE 1200
 #define DEFAULT_AUDIOSINK_SAMPLERATE 44100
 
-- 
1.7.4.1

